tools/theme/text-styles.ts

```diff
@@ -33,6 +43,16 @@ const textStyles = {
     lineHeight: '130%',
     textTransform: 'uppercase',
   },
+  'button-bold': {
+    fontSize: '18px',
+    fontWeight: 'bold',
+    lineHeight: '24px',
+  },
```

Comment:
```suggestion
  'button-semibold': {
    fontSize: '18px',
    fontWeight: 'semibold',
    lineHeight: '24px',
  },
```
Just a small change from design